### ISSUE TITLE
Changing the database name

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Risehat/freemedforms#362]
```
Hi everyone,

When I install the latest freemedforms and want to use a given database name, 
where can I change it? I can only change the user for a db.

Thanks.
```

Original issue reported on code.google.com by `pt.cesar...@gmail.com` on 17 Jul 2014 at 1:27

**Comment by GoogleCodeExporter** [comment]
```
Please explain what you are retint to do. Database names are hardcoded in the 
code. There are no need to rename or manually edit their content. 
```

Original comment by `eric.mae...@gmail.com` on 18 Jul 2014 at 4:49

**Comment by GoogleCodeExporter** [comment]
```
My shared webhosting services gives me an login to my hosting access. Supose my 
login is xyzmyname. If I give remote access to MySQL and introduce the username 
and password setup will automatically create a new database name with 
xyzmyname_somedatabasename and I can't change this at my webhosting service. I 
can change xyzmynam_someothername, not the prefix. It should be changed at the 
software, like I change the username and password.
```

Original comment by `pt.cesar...@gmail.com` on 18 Jul 2014 at 9:02

**Comment by GoogleCodeExporter** [comment]
```
Ok I understand. This feature is not yet available in 0.9.0 but I can add it 
very easily. You should know that there are laws in different countries for 
medical data webhosting. Some are totally not compatible with your objective.
Secondly, FreeMedForms will create more than one database using the following 
scheme

    {yourprefix}_fmf_{database-name}

And this behaviour can not be modified
```

Original comment by `eric.mae...@gmail.com` on 20 Jul 2014 at 8:28

**Comment by GoogleCodeExporter** [comment]
```
Ok. This would be great, because you could put a default value to the prefix, 
and only if the admin wants he can change the 
{yourprefix}_fmf_all-the-default-database-names.

Thank you for your work!
```

Original comment by `pt.cesar...@gmail.com` on 21 Jul 2014 at 8:35

**Comment by GoogleCodeExporter** [comment]
```
This issue was updated by revision 79731f976b3c.
```

Original comment by `eric.mae...@gmail.com` on 23 Jul 2014 at 12:51

**Comment by GoogleCodeExporter** [comment]
```
This issue was updated by revision eb05dec1b7f7.
```

Original comment by `eric.mae...@gmail.com` on 23 Jul 2014 at 12:51

**Comment by GoogleCodeExporter** [comment]
```
This issue was updated by revision 587d6fd125be.
```

Original comment by `eric.mae...@gmail.com` on 24 Jul 2014 at 9:13

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 1817d49101dc.
```

Original comment by `eric.mae...@gmail.com` on 24 Jul 2014 at 9:13
- Changed state: **Fixed**
